src/app/tabs/test-tabs-basic.demo.ts

```diff
@@ -0,0 +1,29 @@
+import { AfterViewChecked, Component, NgZone } from '@angular/core';
+
+/**
+ * This example:
+ * - Shows how to make a simple tab component with an angular template.
+ */
+@Component({
+  selector: 'test-tabs-basic', // tslint:disable-line
+  templateUrl: './test-tabs-basic.demo.html',
+})
+export class TestTabsBasicComponent implements AfterViewChecked {
+  private viewCheckCount = 0;
+  public tabActivatedCount = 0;
+
+  ngAfterViewChecked() {
+    console.log('TestTabsBasicComponent.ngAfterViewChecked called ' + (++this.viewCheckCount) + ' times');
```

Comment:
Do we need to keep these logs in permanent

Reply:
Probably a toast message would be better.

Reply:
Yes, all tests run and samples tested.